usr/sbin/so-clear-backlog

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+. /usr/sbin/so-common
```

Comment:
Pre-elastic systems don't yet have /usr/sbin/so-common.  Maybe we should go ahead and include it in this PR.

Reply:
You're referring to the file itself (including it in the repo), correct?

Reply:
Yes, we'd need to include /usr/sbin/so-common in this package or else /usr/sbin/so-clear-backlog will fail for pre-elastic systems.